README.md

```diff
@@ -50,6 +69,10 @@ Manage URLs.
 
 ## mjs
 
+The mjs files contain code that could benefit from
+some triage;either refactored into `ts` files or removed.
+They can be found under `@bothrs/util/mjs`
```

Comment:
Ik heb een suggestion gedaan in package.json die de mjs files op dezelfde plaats houdt in de build.